spec/spec_helper.rb

```diff
@@ -30,7 +33,7 @@
     #     # => "be bigger than 2"
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
   end
-
+  
```

Comment:
FYI these types of changes cause conflicts. You shouldn't change a line unless you are providing a functional difference in order to make things easier for everyone.